lib/elixir/test/elixir/gen_server_test.exs

```diff
@@ -112,20 +113,67 @@ defmodule GenServerTest do
   @tag capture_log: true
   test "call/3 exit messages" do
     name = :self
-    Process.register self(), name
-    :global.register_name name, self()
+    Process.register(self(), name)
+    :global.register_name(name, self())
     {:ok, pid} = GenServer.start_link(Stack, [:hello])
     {:ok, stopped_pid} = GenServer.start(Stack, [:hello])
     GenServer.stop(stopped_pid)
 
-    assert catch_exit(GenServer.call(name, :pop, 5000)) == {:calling_self, {GenServer, :call, [name, :pop, 5000]}}
-    assert catch_exit(GenServer.call({:global, name}, :pop, 5000)) == {:calling_self, {GenServer, :call, [{:global, name}, :pop, 5000]}}
-    assert catch_exit(GenServer.call({:via, :global, name}, :pop, 5000)) == {:calling_self, {GenServer, :call, [{:via, :global, name}, :pop, 5000]}}
-    assert catch_exit(GenServer.call(self(), :pop, 5000)) == {:calling_self, {GenServer, :call, [self(), :pop, 5000]}}
-    assert catch_exit(GenServer.call(pid, :noreply, 1)) == {:timeout, {GenServer, :call, [pid, :noreply, 1]}}
-    assert catch_exit(GenServer.call(nil, :pop, 5000)) == {:noproc, {GenServer, :call, [nil, :pop, 5000]}}
-    assert catch_exit(GenServer.call(stopped_pid, :pop, 5000)) == {:noproc, {GenServer, :call, [stopped_pid, :pop, 5000]}}
-    assert catch_exit(GenServer.call({:stack, :bogus_node}, :pop, 5000)) == {{:nodedown, :bogus_node}, {GenServer, :call, [{:stack, :bogus_node}, :pop, 5000]}}
+    expected = {
+      :calling_self,
+      {GenServer, :call, [name, :pop, 5000]}
+    }
+
+    assert catch_exit(GenServer.call(name, :pop, 5000)) == expected
```

Comment:
I think it would be preferred to have those as:

```
assert catch_exit(GenServer.call(name, :pop, 5000)) ==
          {:calling_self, {GenServer, :call, [name, :pop, 5000]}}
```

Can you please change this and the ones below accordingly? tks.

Reply:
hey @josevalim , when I ran the formatter it gave the following:
```
    assert catch_exit(GenServer.call(self(), :pop, 5000)) == {
             :calling_self,
             {GenServer, :call, [name, :pop, 5000]}
           }
```
which is why I extracted it into a variable. The same happens with the others in this diff block.

Reply:
Please rebase on latest master and run `make clean compile`. Then if you format it as I suggested, the formatter will keep the formatting. :)

Reply:
ah right! I'll try that